exercicios/para-casa/sistema_de_alunas_patricia.py

```diff
@@ -0,0 +1,134 @@
+#from dataset_alunas import dataset
+dataset = []
+
+def main():
+    # Exibe a mensagem de boas-vindas a loop principal
+    print("\n---  Seja bem vinda a Escola do Reprograma!  ---")
+    print("Sistema de informações de alunas")
+    
+    while True:
+        # Obtém a opção selecionada pelo usuario
+        cod_opcao = obter_opcao()
+
+        #EXECUTA A FUNÇÃO CORREPONDENTE Á OPÇÃO SELECIONADA
+        
+        if cod_opcao == 1: incluir_nova_aluna()
+        elif cod_opcao == 2: consultar_lista_alunas()
+        elif cod_opcao == 3: consultar_faltas_aluna()
+        elif cod_opcao == 4: consultar_notas_aluna()
+        elif cod_opcao == 5: consultar_status_aprovacao()
+        elif cod_opcao == 6: print("Encerrando o programa..."); break
+
+def obter_opcao():
+    #função para obter a opção do usuario
+    codigo_opcao = 0
+
+    while codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+        try:
+
+            #EXIBE O MENU DE OPÇÕES E SOLICITA UMA ESCOLHA DO USUARIO
+
+            codigo_opcao = int(input("\nEscolha uma opção:\n"
+                                    "1 - Incluir uma nova aluna\n"
+                                    "2 - Consultar lista de alunas\n"
+                                    "3 - Consultar faltas da aluna\n"
+                                    "4 - Consultar notas da aluna\n"
+                                    "5 - Consultar status de aprovação\n"
+                                    "6 - Sair do sistema\n"
+                                    "Opção: "))
+                
+            if codigo_opcao not in [1, 2, 3, 4, 5, 6]:
+                print("Opção inválida. Por favor, escolha uma opção válida (1 a 5).\n")
+        except ValueError:
+            print("Entrada inválida. Por favor, digite um número inteiro.\n")
+            
+        return codigo_opcao
+    
+def incluir_nova_aluna():
+    #FUNÇÃO PARA INCLUIR NOVA ALUNA NO DATASET
+
+    nome = input ("Nome da nova aluna:")
+    sobrenome = input ("Sobrenome nova aluna:")
+    turma = input ("Turma:")
+    notas = list(map(float,input("Notas (separadas por espaço):").split()))
+    presencas = list(map(bool,map(int,input("Presenças (1 para presente,0 para ausente, separada por espaço):").split())))
+    participaçao = float(input("Nota de participação:"))    
+    
+    #valida os dados da entrada
+    if not all(0 <= nota <= 10 for nota in notas):
+        print("Erro:Presenças devem ser 1 (prensente)ou 0(ausente).")
+        return
+    if not all(presenca in [True,False] for presenca in presencas):
+        return
+    if not 0 <= participaçao <= 10:
+        print("Erro: Nota de participação deve estar no intervalo de 0 a 10. ")
+        return
+    
+# CRIA UM DICIONÁRIO COM DADOS DA NOVA ALUNA
+    nova_aluna ={
+        "nome": nome,
+        "sobrenome":sobrenome,
+        "turma": turma,
+        "notas": notas,
+        "presencas": presencas,
+        "participacao": participaçao
+
+    }
```

Comment:
O dicionário não segue a estrutura do dataset proposto no exercício.

**Sugestão:**
```suggestion
    dataset[(nome, sobrenome)] = {
        "Turma": turma,
        "Notas": notas,
        "Presença": presencas,
        "Participação": participacao
    }
```